### PR TITLE
Update-deps-2022-05-18

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Update Backend](https://trello.com/c/qUY6dFc8/775-update-backend)

File: operators/src/mock/mock_raster_source.rs [hunk 1]
```diff
@@ -171,7 +171,7 @@ impl<T: Pixel> OperatorDatasets for MockRasterSource<T> {
 ///
 /// TODO: implementation is done with `paste!`, but we can use `core::concat_idents` once its stable
 ///
-/// ```no_run
+/// ```compile_fail
```
Comment (michaelmattig) [inline operators/src/mock/mock_raster_source.rs:174]: wouldn't 

>\`\`\`ignore

be more suitable?

File: services/Cargo.toml [hunk 2]
```diff
@@ -66,13 +66,13 @@ strum = { version = "0.24", features = ["derive"] }
 time = "0.3"
 tokio = { version = "1.15", features = ["macros", "fs", "signal", "sync", "rt-multi-thread"] }
 tokio-util = { version = "0.7", features = ["codec"] }
-tonic = { version = "0.6", features = ["tls", "tls-roots"] }
+tonic = { version = "0.6", features = ["tls", "tls-roots"] } # must be compatible with `scienceobjectsdb_rust_api`
```
Comment (michaelmattig) [inline services/Cargo.toml:69]: couldn't they just reexport the dependency and us using that?

File: services/src/lib.rs [hunk 3]
```diff
@@ -15,6 +15,7 @@
     clippy::option_if_let_else,
     clippy::similar_names,
     clippy::single_match_else,
+    clippy::trait_duplication_in_bounds, // TODO: reactive when not buggy
```
Comment (michaelmattig) [inline services/src/lib.rs:18]: typo?

File: operators/src/lib.rs [hunk 4]
```diff
@@ -15,6 +15,7 @@
     clippy::option_if_let_else,
     clippy::similar_names,
     clippy::single_match_else,
+    clippy::trait_duplication_in_bounds, // TODO: reactive when not buggy
```
Comment (michaelmattig) [inline operators/src/lib.rs:18]: typo?

Comment (ChristianBeilschmidt) [reply operators/src/lib.rs]: [6e1236e](https://github.com/geo-engine/geoengine/pull/539/commits/6e1236e89fe46e67a2640b135057511887d64de4)

File: services/src/lib.rs (hunk 3 above)
Comment (ChristianBeilschmidt) [reply services/src/lib.rs]: [6e1236e](https://github.com/geo-engine/geoengine/pull/539/commits/6e1236e89fe46e67a2640b135057511887d64de4)

File: operators/src/mock/mock_raster_source.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply operators/src/mock/mock_raster_source.rs]: [d2931b8](https://github.com/geo-engine/geoengine/pull/539/commits/d2931b84c4c0ab87267819a4bde9f8147bed727d)

File: services/Cargo.toml (hunk 2 above)
Comment (ChristianBeilschmidt) [reply services/Cargo.toml:69]: Maybe. But this is the reason why we can't update it right now.

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**